padertorch/contrib/examples/pit/evaluate.py

```diff
@@ -92,6 +54,10 @@
 @ex.config
 def config():
     debug = False
+    db_path = ''
+    if "WSJ0_2MIX" in os.environ:
+        db_path = os.environ.get("WSJ0_2MIX")
+    assert len(db_path) > 0, 'Set path to database Json on the command line or set environment variable WSJ0_2MIX'
```

Comment:
You can raise a `sacred.utils.MissingConfigError` here to make clear that this config value is not set. Not sure which one is clearer to people that don't know sacred.

Reply:
I think it's nicer to do it as it is, for now, as you can understand the code in this way without knowing sacred